geps/gep-1924.md

```diff
@@ -0,0 +1,76 @@
+> The "service backend" refers to the target of a service. In Service, this is Endpoints (or EndpointSlice). 
+
+Because the Service resource’s roles are split in this way, it is both a valid parentRef and backendRef for a given xRoute resource. However, Service is not the only resource that can fulfill the frontend role. While the Gateway API spec couldn’t possibly enumerate every existing (and future) frontend-like resource, it can specify a subset of resources that implementations MUST support as parentRefs under as a part of core conformance. Meshes MAY support other implementation-specific resources as parentRefs. The spec maintainers also reserve the right to add additional resources to core conformance as the spec evolves.
+
+## Core Conformance
+
+At present (v0.7.0+), there is only 1 parentRef resources that is under core conformance: Service. The semantics of the Service binding can be found in GEP 1426. 
+
+## Extended Conformance
+
+In addition to Service, there are other optional parentRef resources that, if used by implementations, MUST adhere to the spec’s prescriptions. At the time of writing (v0.70), there are two resources in extended conformance: ServiceImport (part of the mcs-api, currently in alpha) and Gateway . The semantics of ServiceImport parentRef binding can be found in GEP 1748 (Note: Headless ServiceImports are out of scope and not currently a part of the spec). The semantics for Gateways in the context of GAMMA implementations are laid out below:
+
+### `Gateways`
+
+There has been much discussion around cluster local Gateways (i.e. Gateways not associated with a traditional load balancer). While there are various potential UX impairments (e.g. what’s the difference between a GAMMA HTTPRoute with a Gateway parentRef and an ingress implementation’s HTTPRoute?), there is no technical reason why a Gateway cannot be a valid GAMMA parentRef. In this case, the mesh implementation MUST use the ClusterIP of the Gateway for traffic matching (but not necessarily to the exclusion of the External IP).
+
+### Why not `IPAddress`
+
+In Kubernetes 1.27, there will be a new IPAddress resource added to networking.k8s.io/v1alpha1 as part of [KEP 1880](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1880-multiple-service-cidrs#proposal). Naturally, it makes sense to examine whether or not this new resource makes sense as a GAMMA aware parentRef. At first glance, IPAddress seems to be an appropriate abstraction for the “frontend” role we’ve been discussing; every Kubernetes Service is accessed over the network via one of its ip addresses. Furthermore, the fact that the Service resource auto-creates an IPAddress is encouraging. However, the fact that the name of the IPAddress is simply the decimal/hex ip address and not a human-readable Service name makes the UX untenable as a parentRef.
```

Comment:
I agree with this, generally. however, I would say that it makes a _horrendous UX_ to be a parent parentRef, but its not inherently _wrong_. It meets all the properties we need.

In fact, a hypothetical alternative implemented where the `name` was something stable/human readable, it would be perfectly fine.

So I think we should not make it an official thing supported, but we also don't need to do anything like say "You MUST not use IPAddress as a parentRef". If you can call it and attach routes to it, its a possible parentRef.

Reply:
Agreed - I don't think anything here says "you MUST" not use it, but I can preemptively clear this up

Reply:
Agreed. Anything with a FQDN or IP is a valid parentRev - if the WG chooses to.

That doesn't mean we MUST - we can take UX and practical considerations into account.

Reply:
Added `However, `IPAddress` is NOT disallowed; implementations may use it if they wish.`